clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -26,6 +26,7 @@ public class OffsetSpec {
     public static class EarliestSpec extends OffsetSpec { }
     public static class LatestSpec extends OffsetSpec { }
     public static class MaxTimestampSpec extends OffsetSpec { }
+    public static class EarliestLocalTimestampSpec extends OffsetSpec { }
```

Comment:
Can we rename `EarliestLocalTimestampSpec` to `EarliestLocalSpec` similar to earliest and latest?